rpc/commands/lib.proto

```diff
@@ -261,6 +266,8 @@ message Library {
     LibraryLocation location = 24;
     // The library format type.
     LibraryLayout layout = 25;
+    // The example sketches provided by the library
+    repeated string examples = 26;
```

Comment:
Please consider moving the `examples` from `Library` to `LibraryRelease`.

For me, a `Library` is the unique identifier (with its `name`) and it has `1..*` `LibraryReleases`.

A `Library` can have, for example, two releases, `1.0.0` and `1.1.0`.

The `1.0.0` version can have two examples:
```
[
  '/path/to/example/A',
  '/path/to/example/B'
]
```

And the latest, `1.1.0` `LibraryRelease` version can have an update, so it has comes with three examples:
```
[
  '/path/to/example/A',
  '/path/to/example/B',
  '/path/to/example/NewFeatureFrom_1_1_0',
]
```

Possible related: #928

Reply:
+1 for merging this PR as-is; we can handle #928 later together with the currently unnecessary `instance`.